docs/articles/documentation/using-testcafe/programming-interface/BrowserConnection.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: BrowserConnection Class
+permalink: /documentation/using-testcafe/programming-interface/BrowserConnection/
+---
+# BrowserConnection Class
+
+Connection to a remote browser.
+
+Created by the [testCafe.createBrowserConnection](/testcafe/documentation/using-testcafe/programming-interface/TestCafe/#createBrowserConnection) function.
+
+#### Example
+
+```js
+let createTestCafe   = require('testcafe');
```

Comment:
@VasilyStrelyaev all variables declared here should be `const` since they are immutable. Alignment should be done only in blocks without blank line. Separate `require`'s from other declarations.  So:
```js
let createTestCafe   = require('testcafe');
let testCafe         = await createTestCafe('localhost', 1337, 1338);

let remoteConnection = testcafe.createBrowserConnection();
```
should be:
```js
const createTestCafe = require('testcafe');

const testCafe         = await createTestCafe('localhost', 1337, 1338);
const remoteConnection = testcafe.createBrowserConnection();
```